t/058-tcp-socket.t

```diff
@@ -3808,3 +3808,131 @@ received:
 received: truefalsenil
 --- no_error_log
 [error]
+
+
+
+=== TEST 64: *b pattern for receive
+--- config
+    server_tokens off;
+    location = /t {
+        set $port $TEST_NGINX_SERVER_PORT;
+        content_by_lua_block {
+            local sock = ngx.socket.tcp()
+            sock:settimeout(500)
+            assert(sock:connect("127.0.0.1", ngx.var.port))
+            local req = {
+                'GET /foo HTTP/1.0\r\n',
+                'Host: localhost\r\n',
+                'Connection: close\r\n\r\n',
+            }
+            sock:send(req)
```

Comment:
Better to check the return values of `send()` and handle any potential errors? Please fix other similar places (if any).